sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient_private.cs

```diff
@@ -24,62 +24,62 @@ public partial class TextAnalyticsClient
         private const string ModelVersion = "model-version";
 
         #region Detect Language
-        private static async Task<Response<DetectLanguageResultCollection>> CreateDetectLanguageResponseAsync(Response response, CancellationToken cancellation)
+        private static async Task<Response<DetectLanguageResultCollection>> CreateDetectLanguageResponseAsync(Response response, Dictionary<string, int> idToIndexMap, CancellationToken cancellation)
         {
-            DetectLanguageResultCollection result = await TextAnalyticsServiceSerializer.DeserializeDetectLanguageResponseAsync(response.ContentStream, cancellation).ConfigureAwait(false);
+            DetectLanguageResultCollection result = await TextAnalyticsServiceSerializer.DeserializeDetectLanguageResponseAsync(response.ContentStream, idToIndexMap, cancellation).ConfigureAwait(false);
             return Response.FromValue(result, response);
         }
 
-        private static Response<DetectLanguageResultCollection> CreateDetectLanguageResponse(Response response) =>
-            Response.FromValue(TextAnalyticsServiceSerializer.DeserializeDetectLanguageResponse(response.ContentStream), response);
+        private static Response<DetectLanguageResultCollection> CreateDetectLanguageResponse(Response response, Dictionary<string, int> idToIndexMap) =>
```

Comment:
Nit: even for private members, if you have no reason to take a concrete classes (example, only `Dictionary<,>` exposes it's key comparer, unfortunately), I'd still use least-derived type to make refactoring easier later.